docs/reference/query.rst

```diff
@@ -163,18 +163,24 @@ and day are optional. If you leave out the day, for example, you will get
 matches for the whole month.
 
 You can also use relative dates to the current time.
-Relative dates begin by the ``@`` character, followed by an optional ``+`` or
-``-`` sign that will increment or decrement now, followed by the time quantity
-that will be represented as an integer followed by either ``d`` for days, ``w``
-for weeks, ``m`` for months and finally ``y`` for year.
+A relative date begins by ``@`` and has to follow the pattern_dq format
+``@([+|-]?)(\d+)([y|m|w|d])``
```

Comment:
This is a very unwieldy piece of information. Maybe something more like "… and looks like ``@+3y``." E.g., give an example of the pattern instead of the regex definition of what a pattern has to adhere to («show, don't tell»). This is end-user documentation, not developer documentation. End users should not be expected to be able to read regular expressions. Besides, you explain (almost) each part of the pattern excellently below.